src/develop/blend_gui.c

```diff
@@ -1616,6 +1908,18 @@ void dt_iop_gui_init_blendif(GtkBox *blendw, dt_iop_module_t *module)
       gtk_box_pack_start(GTK_BOX(bd->blendif_box), GTK_WIDGET(slider_box), TRUE, FALSE, 0);
     }
 
+    bd->channel_boost_factor_slider = dt_bauhaus_slider_new_with_range(module, 0.0f, 3.0f, .02f, 0.0f, 3);
+    dt_bauhaus_slider_set_format(bd->channel_boost_factor_slider, _("%.2f EV"));
+    dt_bauhaus_widget_set_label(bd->channel_boost_factor_slider, NULL, _("boost factor"));
```

Comment:
dt_bauhaus_widget_set_label now needs to receive an english label, so use N_("boost factor")

Reply:
also, section is N_("blend")

Reply:
I have fixed that. And one other instance. Thanks for catching !